configure.ac

```diff
@@ -72,6 +71,33 @@ AC_ARG_ENABLE(
 AS_IF([test "x$esapi_sf" = "xyes"],
        [do_esapi_manage_flags])
 
+AC_ARG_WITH(
```

Comment:
Shouldn't this be AC_ARG_ENABLED with auto being default ?

Reply:
I initially wen't that route, but decided to go to "with" option, since it's --with-PACKAGE or --without-PACKAGE and this completely removed the FAPI package dependency.

But I'd be open to arguments convincing me otherwise. It's also a feature... so I could see AC_ARG_ENABLED as well.

Reply:
I think I convinced myself to go back to AC_ARG_ENABLED